incubator/azuremonitor-containers/values.yaml

```diff
@@ -6,9 +6,9 @@
 ## ref: https://github.com/Microsoft/OMS-docker/tree/ci_feature_prod
 omsagent:
   image:
-    tag: "ciprod09262019"
+    tag: "ciprod10112019"
```

Comment:
we found a regression in this image. Please DONT MERGE this. Check with either me or Vishwa

Reply:
> we found a regression in this image. Please DONT MERGE this. Check with either me or Vishwa

@r-dilip, updated to latest version which has fix as we discussed.